packs/data/feats.db/self-destruct.json

```
@@ -0,0 +1,45 @@
+{
+    "_id": "7aHYM9fawA3PQwtM",
+    "data": {
+        "actionCategory": {
+            "value": "offensive"
+        },
+        "actionType": {
+            "value": "reaction"
+        },
+        "actions": {
+            "value": null
+        },
+        "description": {
+            "value": "<p><strong>Trigger</strong> Your reanimated clockwork companion is reduced to 0 Hit Points.</p>\n<p><strong>Requirements</strong> You're within 60 feet of your reanimated clockwork companion.</p>\n<hr />\n<p>You've installed a self-destruct device within your construct companion, which you can trigger with a special device from up to 60 feet away. When you activate the device, your reanimated construct companion explodes, dealing [[/r {2d6}[fire]]]{2d6 fire damage} per level to all creatures in a @Template[type:emanation|distance:30]{30-foot radius}. Creatures caught in the blast can attempt a @Check[type:reflex |dc:resolve(@actor.attributes.classOrSpellDC.value)] save, with a DC equal to the higher of your class DC or spell DC.</p>"
```

```suggestion
            "value": "<p><strong>Trigger</strong> Your reanimated clockwork companion is reduced to 0 Hit Points.</p>\n<p><strong>Requirements</strong> You're within 60 feet of your reanimated clockwork companion.</p>\n<hr />\n<p>You've installed a self-destruct device within your construct companion, which you can trigger with a special device from up to 60 feet away. When you activate the device, your reanimated construct companion explodes, dealing [[/r {2d6}[fire]]]{2d6 fire damage} per level to all creatures in a @Template[type:emanation|distance:30]{30-foot radius}. Creatures caught in the blast can attempt a @Check[type:reflex|dc:resolve(@actor.attributes.classOrSpellDC.value)] save, with a DC equal to the higher of your class DC or spell DC.</p>"
```

Add a rollable after the text for the leveled damage